ci/Jenkinsfile-linux-integration

```diff
@@ -233,18 +207,16 @@ builders['linux-tests'] = testBuilder('linux', 'py35', '/workspace')({
         sh '''
            rm -rf ~/.dcos; \
            grep -q "^.* dcos.snakeoil.mesosphere.com$" /etc/hosts && \
-           sed -iold "s/^.* dcos.snakeoil.mesosphere.com$/${DCOS_URL} dcos.snakeoil.mesosphere.com/" /etc/hosts || \
-           echo ${DCOS_URL} dcos.snakeoil.mesosphere.com >> /etc/hosts'''
+           sed -iold "s/^.* dcos.snakeoil.mesosphere.com$/${DCOS_IP} dcos.snakeoil.mesosphere.com/" /etc/hosts || \
+           echo ${DCOS_IP} dcos.snakeoil.mesosphere.com >> /etc/hosts'''
 
         dir('dcos-cli/cli') {
             sh '''
                export PYTHONIOENCODING=utf-8; \
-               export DCOS_CONFIG=tests/data/dcos.toml; \
-               chmod 600 ${DCOS_CONFIG}; \
-               echo dcos_acs_token = \\\"${DCOS_ACS_TOKEN}\\\" >> ${DCOS_CONFIG}; \
-               cat ${DCOS_CONFIG}; \
-               unset DCOS_URL; \
-               unset DCOS_ACS_TOKEN; \
+               make binary; \
+               dist/dcos cluster setup dcos.snakeoil.mesosphere.com --insecure --username=${DCOS_ADMIN_USERNAME} --password-env=DCOS_ADMIN_PASSWORD; \
```

Comment:
I'd rather leave this as it was before so that we can test that config migration works properly.

Reply:
on the other hand right now we don't have tests for `dcos cluster setup`, it also simplifies the jenkinsfile.

Reply:
Given the number of different scenarios for the migration, I can probably create a dedicated test suite for these, WDYT? It seems hard to add such tests in our integration suite, as it assumes an already-setup cluster, some env variables already set.

Reply:
I've added integration tests, cf. 5a4d013c169e9c1be8ddde8c74834a34d65959f5